src/main/webapp/navigations.jsp

```diff
@@ -1,21 +1,11 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <link rel="stylesheet" href="/css/main.css">
-</head>
-<body>
-  <nav>
-    <a id="navTitle" href="/">CodeByter's Chat App</a>
-    <a href="/conversations">Conversations</a>
-    <% if (request.getSession().getAttribute("user") != null) { %>
-      <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
-      <a href="/users/<%= request.getSession().getAttribute("user")%>">My Profile</a>
-      <a href="/logout.jsp">Logout</a>
-    <% } else { %>
-      <a href="/login">Login</a>
-    <% } %>
-    <a href="/activityfeed">ActivityFeed</a>
-    <a href="/about.jsp">About</a>
-  </nav>
-</body>
-</html>
\ No newline at end of file
+<a id="navTitle" href="/">CodeByter's Chat App</a>
```

Comment:
You can leave the <nav> and </nav> tags in here so you don't have to paste them into every file.